lib/link_text.dart

```diff
@@ -15,6 +14,11 @@ import 'schemes.dart';
 import 'tlds.dart';
 
 typedef LinkTapHandler = void Function(Uri);
+typedef TextSpanBuilder = TextSpan Function(
+  String? text,
```

Comment:
if it's optional, why don't you put it inside { }, so that we don't have to write null again and again ?

Reply:
it's related to @drminh2807 's feature